core/primitives-core/src/version.rs

```diff
@@ -266,7 +266,7 @@ impl ProtocolFeature {
 /// Current protocol version used on the mainnet.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-const STABLE_PROTOCOL_VERSION: ProtocolVersion = 69;
+const STABLE_PROTOCOL_VERSION: ProtocolVersion = 81;
```

Comment:
This should not be needed, we have the statelessnet_protocol to do this overwrite. 
Also since stateless net is on 82 we should not be going down from there.

Reply:
Reverted, the reason for this was to keep the changes on top of the other stabilization PR minimal, as this will revert a bunch of other files, but it is ok.

Reply:
ah it's fine if this makes this PR easier but then I would at least make it match statelessnet_protocol